wisegui/src/lib.rs

```diff
@@ -40,13 +40,14 @@ impl Font {
             }
         }
 
-        Font {
-            chars: chars,
-        }
+        Font { chars: chars }
```

Comment:
Man, rustfmt _really_ went to town on this, which is a bit unfortunate, as it's quite tangled with the rest of the commit contents as-is, and I'd argue these changes are completely unrelated.

I have a (bad?) habit of _not_ using rustfmt, so I guess I'm partially to blame here. But I would have much preferred to have the rustfmt changes in a single commit, and then the line drawing stuff on top of that in a separate commit. Would it be possible for you to split these out?

Reply:
I can totally separate that out.